pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -333,7 +333,8 @@ type ScorePlugin interface {
 type ReservePlugin interface {
 	Plugin
 	// Reserve is called by the scheduling framework when the scheduler cache is
-	// updated.
+	// updated. If this method returns a failed Status, the scheduler will call
+	// the Unreserve method for all enabled ReservePlugins.
 	Reserve(ctx context.Context, state *CycleState, p *v1.Pod, nodeName string) *Status
 	// Unreserve is called by the scheduling framework when a reserved pod was
 	// rejected, an error occurred during reservation of subsequent plugins, or
```

Comment:
Instead of "The Unreserve method implementation must be idempotent." To be more explicit, let's say: "Note that Unreserve might be called even if Reserve for the same plugin was not."

Reply:
I think it makes sense to include both statements -- I've added a line to say that.